src/braket/circuits/ascii_circuit_diagram.py

```diff
@@ -288,7 +296,7 @@ def _ascii_diagram_column(
                         else ascii_symbols[item_qubit_index]
                     )
                 elif qubit in control_qubits:
-                    symbols[qubit] = "C"
+                    symbols[qubit] = "C" if map_control_qubit_states[qubit] else "N"
```

Comment:
I think this is a good improvement. Would love to get a second opinion from @speller26 to make sure I'm not missing anything (ie. name conflicts or existing conventions) with this notation

Reply:
Yes, it might be a good time to update to that. Might want to think if that's a breaking change, but I suspect we can assume users use the printed representation purely visually, though it may trigger some test failures for libraries built on Braket

Reply:
Yes, that's so much nicer. I say we do it. We'll need to manually update gates with built-in control qubits as well, like cnot. I will poll the devx team to see if there's alignment